data/lisboa_viva/license-prefix.txt

```diff
@@ -0,0 +1,2 @@
+This software contains data from Opendatasoft. The following terms apply to Opendatasoft:
```

Comment:
Unlike the other agencies, it's not really clear what that data is, maybe something like:

"This software contains Lisbon public transit data from Opendatasoft. The following terms apply to this data:"